.github/workflows/template-main-e2e-test.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   e2e-tests:
     name: Run e2e test
-    runs-on: ARM64
+    runs-on: ubuntu-24.04-arm
```

Comment:
Main e2e should be executed on the ARM64 machine because it executes >20 concurrent e2e tests (it's the runner) against the k8s cluster and probably and we have seen that small machines can be affected. Or we can try to change it and check what happens. @kedacore/keda-core-maintainers @kedacore/keda-core-contributors ?